### PR TITLE
Fix migration docs

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Adding info about migrating ABP device with `ttn-lw-migrate` tool.

#### Notes for Reviewers
@neoaggelos I'm not sure if this can be done. I know that ABP device can be migrated with this tool with active session, but can't it be migrated with the same DevAddr and RX1 Delay using this tool + migrating a gateway?

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by neoaggelos — APPROVED** [review]
LGTM in general. Maybe reword "migrate your ABP end device without active session" as "re-program your ABP device for (TTS v3)"

File: doc/content/getting-started/migrating/migrating-from-v2/migrate-using-migration-tool/establish-new-session.md [hunk 1]
```diff
@@ -58,14 +78,32 @@ $ ttn-lw-migrate application --source ttnv2 "ttn-v2-application-ID" --ttnv2.with
...
+
+{{< tabs/tab "OTAA" >}}
+
 To prevent OTAA device from re-joining the {{% ttnv2 %}} network, the recommended practice is to change the **AppKey** in the {{% ttnv2 %}}. By changing the **AppKey**, the existing session on {{% ttnv2 %}} will not be terminated yet, but the end device will not be able to re-join because the {{% ttnv2 %}} cluster will reject its new Join Requests. 
 
+{{< /tabs/tab >}}
+
+{{< tabs/tab "ABP" >}}
+
+For ABP device, you need to completely delete it from V2, especially because this section assumes that you have not re-programmed it for a new **DevAddr**. Having this device registered in V2 and The Things Stack would introduce some serious conflicts.
```
Comment (laurensslats) [inline doc/content/getting-started/migrating/migrating-from-v2/migrate-using-migration-tool/establish-new-session.md:91]: This part is important right? Let's add the warning tag

```suggestion
{{< warning >}} For ABP device, you need to completely delete it from V2, especially because this section assumes that you have not re-programmed it for a new **DevAddr**. Having this device registered in V2 and The Things Stack would introduce some serious conflicts. {{</ warning >}}
```

File: doc/content/getting-started/migrating/migrating-from-v2/migrate-using-migration-tool/establish-new-session.md [hunk 2]
```diff
@@ -83,3 +121,13 @@ Since we assume that you have not migrated your gateway from {{% ttnv2 %}} yet,
...
 
 {{< note >}} Even if you manage to get your end device traffic routed to {{% tts %}} via Packet Broker, we still recommend you get in touch with your local The Things Network community and coordinate the migration of gateways, so you do not loose the LoRaWAN network coverage. See how to [migrate your gateway to {{% tts %}}]({{< ref "/getting-started/migrating/gateway-migration" >}}). {{</ note >}}
+
+{{< /tabs/tab >}}
+
+{{< tabs/tab "ABP" >}}
+
+## Next Step - Migrate Gateways
+
+This section implies that you are keeping the **DevAddr** and **RX1 Delay** values from {{% ttnv2 %}}, which means Packet Broker will not be able to route the traffic between your end device and {{% tts %}} neither properly, nor on time. Hence, you will have to [migrate your gateway]({{< ref "/getting-started/migrating/gateway-migration" >}}) as well.
```
Comment (laurensslats) [inline doc/content/getting-started/migrating/migrating-from-v2/migrate-using-migration-tool/establish-new-session.md:131]:
```suggestion
This section implies that you are keeping the **DevAddr** and **RX1 Delay** values from {{% ttnv2 %}}, which means Packet Broker will not be able to route the traffic between your end device and {{% tts %}}. Hence, you will have to [migrate your gateway]({{< ref "/getting-started/migrating/gateway-migration" >}}) as well. 
```

**Review by laurensslats — APPROVED** [review]
Looking good. Pls apply my 2 suggestions before merging.

**Comment by nejraselimovic** [comment]
> LGTM in general. Maybe reword "migrate your ABP end device without active session" as "re-program your ABP device for (TTS v3)"

I'm confused, what are you referring to exactly? If you are talking about migrating with the migration tool - afaik, if you re-program the device for V3 it is no longer connected to V2, hence, you can't do anything with the migration tool in terms of migration since migration tool exports a description from V2 (that gets old because your device no longer uses those parameters). But if you don't re-program the device, you can migrate it to V3 using migration tool without security keys + you must migrate the gateway too (this use case is what I'm referring to when talking about migration without session for ABP devices using migration tool). It might be wrong to call this migration without active session (because you're not establishing a new session either), but that's a bit of gray area for me.

**Comment by nejraselimovic** [comment]
@johanstokking can you please take a quick look at this and let me know if I understood this correctly. 

The question is - if I don't re-program the ABP device, can I migrate it from V2 using `ttn-lw-migrate` with a `--ttnv2.with-session=false` flag? If this is possible, is it completely false to refer to this use case as migrating "without active session"? 

Also, if I re-program the ABP device, I cannot use the `ttn-lw-migrate` tool to migrate it from V2 because then the description of the device stored in V2 and the parameters hardcoded in the device don't match anymore, right?
